service/src/intTest/java/uk/nhs/adaptors/gp2gp/ehr/EhrExtractStatusServiceIT.java

```diff
@@ -305,7 +306,7 @@ private void addInProgressTransfer(String conversationId, List<EhrExtractStatus.
                 .build())
             .gpcAccessStructured(EhrExtractStatus.GpcAccessStructured.builder()
                 .accessedAt(FIVE_DAYS_AGO)
-                .objectName(generateRandomUppercaseUUID() + ".json")
+                .objectName(generateRandomUppercaseUUID() + JSON_SUFFIX)
```

Comment:
I think maybe having a separate method which simply returns `generateRandomUppercaseUUID() + JSON_SUFFIX` - this way you won't need to repeat `JSON_SUFFIX`.

Reply:
ok, yes, I will crate a new method